hyperspy/signal.py

```diff
@@ -39,12 +39,13 @@
 import traits.api as t
 from tlz import concat
 
-import hyperspy
 from hyperspy.axes import AxesManager
 from hyperspy.api_nogui import _ureg
+from hyperspy.misc.array_tools import rebin as hsrebin
```

Comment:
Again, everything else is imported by `import from` (and it might not always be clear from which module of hyperspy they come from), except for these two functions. However, `save` and `rebin` can not be imported without giving them new names.

Referencing any hyperspy function with their full path, will not make the code more readable either, as it would blow up the code by many long function names.

I could now:
a) just leave it as it was originally
b) give the functions more instructive new names, such as `hs_array_rebin`

Reply:
Here I renamed the imports with more instructive names. Everywhere else, I opted for the more general import (except where `dask.array` was extensively used).